src/client/blobs.rs

```diff
@@ -0,0 +1,179 @@
+use super::BlobScanClient;
+use alloy_chains::Chain;
+use serde::{ser::StdError, Deserialize, Serialize};
+use std::fmt;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct BlobListResponse {
+    blobs: Vec<Blob>,
+    #[serde(rename = "totalBlobs")]
+    total_blobs: u32,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Blob {
+    #[serde(rename = "versionedHash")]
+    versioned_hash: String,
+    commitment: String,
+    proof: String,
```

Comment:
i think ideally we have these strongly typed, i.e. convert from str into bytes with serde